helpers/hello_world_no_os/config/custom_config_ram.h

```diff
@@ -0,0 +1,58 @@
+/**
+ ****************************************************************************************
+ *
+ * @file custom_config_ram.h
```

Comment:
The MIT License (MIT)

Reply:
Could you provide more info on this comment? I do not seen such 'contents' in other SDK files.